src/components/extensive/Modal/ModalAdd.tsx

```diff
@@ -67,6 +71,54 @@ const ModalAdd: FC<ModalAddProps> = ({
       setFile(files);
     }
   }, [files, setFile]);
+
+  const handleFileChange = (files: File[]) => {
+    const formatFile = (file: File): UploadedFile => ({
+      title: file.name,
+      file_name: file.name,
+      mime_type: file.type,
+      collection_name: "storybook",
+      size: file.size,
+      url: "https://google.com",
+      created_at: "now",
+      uuid: file.name,
+      is_public: true,
+      rawFile: file
+    });
+
+    const acceptedFileTypes = acceptedTYpes
+      ? acceptedTYpes
```

Comment:
Please fix the spelling (`acceptedTypes`). Also I'm curious about the join + split... isn't that a NOOP? Or are you doing it to effectively cast the values in the starting array to string? If so, I'd do this in the examples below instead of doing multiple array operations.
```
type => `${type}`.trim() 
```

🎨 I'd skip the ternary here:
```
const acceptedFileTypes = acceptedTypes?.map(type => type.trim()) ?? []
```
or 
```
const acceptedFileTypes = (acceptedTypes ?? []).map(type => type.trim())
```

Reply:
spelling changed in all related files and accepted the suggestion.